### ISSUE TITLE
Per world cooldowns aren't applied

### ISSUE THREAD

**Issue by christaylor201** [SuperRonanCraft/BetterRTP#97]
**Plugin Version**:
3.4.2-1

**Describe The Bug**:
Per world cooldowns aren't applied, players can randomly teleport as if no cooldowns have been set.

**How To Reproduce**:
1. Enable the `Settings.Cooldown.PerWorld` option.
2. Set a different cooldown for each custom world.
3. Observe the mentioned issue.

**Expected Behavior**:
Per world cooldowns should be applied.

**Screenshots/Error Log**:
None.

**Additional Context**:
Here's our [configuration](https://gist.github.com/christaylor201/d735ea46268c7a95f7dbdb1ba50862d8) and [the output](https://gist.github.com/christaylor201/1eaea48ca2c9c0a80bf9e3e0bcb81668) of the `/rtp info` command. As you can see, each world has a different non-zero cooldown. However, the parts of the output containing `Cooldown: 0N/A` don't seem right to me; are the cooldowns loaded correctly?

**Comment by SuperRonanCraft** [comment]
Hey mate, so the way you executed /rtp info is viewing it with the current permissions of the player executing, so most likely you are bypassing cooldowns so they are not being applied, the `N/A` shows up is when the command is not executed by a player, so a permission check is not performed.

I would suggest performing `/rtp info world <player you want to check>` and see what info shows up, and check if players are allowed to bypass cooldowns with the `betterrtp.bypass.cooldown` permission

**Comment by christaylor201** [comment]
I originally ran the `/rtp info` command from the console, but the output of the `/rtp info world <player>` command is the same, only the number of queued positions differs.

I also checked the given permission, and none of our players inherit it.

**Comment by SuperRonanCraft** [comment]
Okay, then I would recommend enabling `Debugger` in the config and get me a log after executing `/rtp reload`

**Comment by christaylor201** [comment]
[Here](https://gist.github.com/christaylor201/08ec54c97401b5e6fc5536a12677e56a) you are.

**Comment by SuperRonanCraft** [comment]
Just uploaded a new patch that should fix this issue! This problem was actually very hidden and required only one line of code, but worked it out with someone and we got it fixed! Grab the new version here https://www.spigotmc.org/resources/betterrtp.36081/update?update=453764

**Comment by christaylor201** [comment]
Unfortunately, this only partially fixed the issue – cooldowns are now applied per world, but instead of the world-specific durations, the global one is used for all.

**Comment by iUndef** [comment]
I've updated to the latest release and am also seeing this issue. I have a custom world set, including the setting 'Cooldown: 900'. I also have the 'Cooldown' section setting 'PerWorld' set to true. However every world just uses the default cooldown time that's set in the 'Cooldown' section. I've attached my current config for reference.
[kohirtp.txt](https://github.com/SuperRonanCraft/BetterRTP/files/8560296/kohirtp.txt)
